lib/utils/package-up.js

```diff
@@ -16,13 +16,17 @@ const { fileURLToPath } = require('url');
 /**
  * Inlined version of the package "package-up" (ESM only).
  *
- * @param {string} cwd The directory to start searching from.
+ * @param {{cwd: string}} options The directory to start searching from.
```

Comment:
Shouldn't we keep the previous syntax? Here the description "The directory (...)" refers to the `options` object, and not to `cwd` right?

Or we can remove the description, since it's super internal, I don't really have a strong opinion on this.

Reply:
The previous syntax describes a `cwd` argument which does not exist (there is a `cwd` property in the first argument instead).

Reply:
I updated the JSDoc to use the alternative syntax allowing to put a description on properties (this was already used later in the file)